src/docs/premium.mdx

```diff
@@ -7,26 +7,26 @@ sidebar_label: Premium
 custom_edit_url: https://github.com/EarthMC/earthmc-docs/blob/main/src/docs/premium.mdx
 ---
 # Premium
-Premium is a paid rank that helps fund the server and its development. It works like patreon if you're familiar with that. Subscribers pay $7.99 per month and get awesome perks as thanks for helping the server out. 
+Premium is a paid rank that helps fund the server and its development. It works like Patreon if you're familiar with that. Subscribers pay $7.99 per month and get awesome perks as thanks for helping the server out. 
 
-One of the many great benefits with premium is **queue priority** which lets subscribers save time by moving past everyone without premium in queue. Premium also gives subscribers a **gold name** in chat, ability to **change map claim color** and **exclusive access to beta servers**.
+One of the many great benefits of premium is **queue priority** which lets subscribers save time by moving past everyone without premium in the queue. Premium also gives subscribers a **gold name** in chat, the ability to **change map claim color**, and **exclusive access to beta servers**.
 Mystery crates and exclusive collectibles are other perks obtained with premium.
 
-[Here is the store page](https://store.earthmc.net/category/premium) if you are interested in subscribing. Supported payment methods are **PayPal**, **Stripe** and **Trustly**.
+[Here is the store page](https://store.earthmc.net/category/premium) if you are interested in subscribing. Supported payment methods are **PayPal**, **Stripe**, and **Trustly**.
 
 ## Premium commands
 | Command        | Description
 | ------------- |-------------
 | /hat   | Use any item as a hat
-| /fly      | Fly in hub
+| /fly      | Fly in the hub
 | /nick | Change name in chat
 | /party | Private party, share mcMMO XP with your friends
 | /seen | See when someone was last online
 | /qs staff | Hire staff for your shop
-| /discord link | Pink username on emc discord
+| /discord link | Pink username on EMC discord
 | /ch j premium | Join a private premium-only chat
 | /emcmap nationcolor | Change dynmap territory fill color
-| /emcmap nationoutline | Change dynmap territory border color
+| /emcmap nationoutline | Change dynmap territory border-color
 
 ### Cancel
-You can easily cancel your subscription on your PayPal account dashboard or by clicking a link in your purchase confirmation email. There are no cancellation fees. Start or stop your subscription access at any time. If you need help to manually cancel the subscription, reach out to contact@earthmc.net.
+You can easily cancel your subscription on your PayPal account dashboard or click a link in your purchase confirmation email. There are no cancellation fees. Start or stop your subscription access at any time. If you need help canceling the subscription manually, reach out to contact@earthmc.net.

```

Comment:
"by clicking" is more correct here. Easiest way to check is to split the sentence at the conjunction and check both possibilities:

✅ You can easily cancel your subscription on your PayPal account dashboard.  
✅ You can easily cancel your subscription by clicking a link in your purchase confirmation email.  
❌ You can easily cancel your subscription click a link in your purchase confirmation email.